File: apps/android/app/src/main/java/com/roundsloyalty/app/data/models/LoyaltyBalance.kt
package com.roundsloyalty.app.data.models

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class LoyaltyBalance(
    val id: String,
    @SerialName("customer_id") val customerId: String,
    @SerialName("vendor_id") val vendorId: String,
    val stamps: Int = 0,
    val points: Int = 0,
)

File: apps/ios/RoundsLoyalty/Models/LoyaltyBalance.swift
import Foundation

struct LoyaltyBalance: Codable, Identifiable {
    let id: UUID
    let customerId: UUID
    let vendorId: UUID
    let stamps: Int
    let points: Int

    enum CodingKeys: String, CodingKey {
        case id
        case customerId = "customer_id"
        case vendorId = "vendor_id"
        case stamps
        case points
    }
}
